packages/lib/types/index.d.ts

```diff
@@ -245,6 +245,21 @@ declare interface RemotesConfig {
    * from
    */
   from?: 'vite' | 'webpack'
+
+  /**
+   * The number of milliseconds to wait between import retries. Defaults to 500ms
+   */
+  importRetryBackoff?: number
```

Comment:
could we do exponential backoff here rather than a fixed constant backoff? i think it makes more sense in cases like this with network requests. we want to retry ASAP, especially since in our case we know the failures often come from a re-deployment which should be immediately available, but we also don't want to hammer the server if one retry doesn't cut it. so an exponential backoff i think would be a little more appropriate to get that ASAP retry for the best user experience.

it could also provide a smaller public API, just providing the retry count.